src/vs/workbench/parts/output/electron-browser/outputServices.ts

```diff
@@ -323,19 +282,26 @@ class OutputFileListener extends Disposable {
 
 	constructor(
 		private readonly file: URI,
+		private readonly fileService: IFileService
 	) {
 		super();
 	}
 
 	watch(): void {
 		if (!this.watching) {
-			this.disposables.push(watchFile(this.file.fsPath, () => this._onDidChange.fire()));
+			this.fileService.watchFileChanges(this.file);
```

Comment:
Maybe add this to the list of `Disposable`: `toDisposable(() => this.fileService.unwatchFileChanges....)`?

Reply:
@bpasero `unwatch` is called in `dispose` method.

Reply:
@sandy081 yeah its fine as long as `watch` and `unwatch` are always called in sync.